packages/core/core-flows/src/order/workflows/create-return.ts

```diff
@@ -339,7 +339,21 @@ export const createReturnOrderWorkflow = createWorkflow(
       prepareFulfillmentData
     )
 
-    createFulfillmentWorkflow.runAsStep(fulfillmentData)
+    const returnFulfillment =
+      createFulfillmentWorkflow.runAsStep(fulfillmentData)
```

Comment:
Q: I think you have changed it in the other pr right?

Reply:
yes, I did